components/input.css

```diff
@@ -0,0 +1,84 @@
+.input-field{
+    width: 70%;
+    padding: 0.5rem 1rem;
+    border-radius: 5px;
+    outline: none;
+    border: 1px solid black;
+}
+
+.disabled-input{
+    pointer-events:none;
+    background-color: rgb(231, 231, 231);
+}
+
+.input-err{
+    width: 70%;
+}
+
+.input-err .input-field{
+    width: 100%;
+    border: 1px solid red;
+}
+
+.err-msg{
+    font-size: 0.8rem;
+    color: red;
+    margin-top: 2px;
+}
+/* 
+input[type="radio"] {
+    color: currentColor;
+    width: 1.5rem;
+    height: 1.5rem;
+    border: 0.15em solid currentColor;
+  }
+
+  input[type="radio"]:checked{
+    transition: all 0.8s ease-out;
+    border-color: black;
+    border-width: 5px;
+
+  } */
+
+
+  .radio, .checkbox {
+    appearance: none;
+    margin: 0 0.5rem;
+  }
+  
+  .radio:checked, .checkbox:checked {
+    border-color: var(--primary-alert-bg-color);
+  }
+
+  .circle {
+    border: 7px solid var(--primary-color);
+    border-radius: 100%;
+    background-color: var(--primary-color);
+  }
+
+  .square{
+    border: 7px solid var(--primary-color);
+    /* border-radius: 100%; */
+    background-color: var(--primary-color);
+  }
+  
+  .circle:checked, .square:checked {
+    border-width: 5px;
+  }
+
+  .circle:checked, .square:checked {
+    border-width: 5px;
+  }
+
+  .disabled-radio, .disabled-checkbox{
+    pointer-events:none;
+    border-color: rgb(190, 190, 190);
+    background-color: rgb(190, 190, 190);
+  }
+  
+  .circle:checked:before, .square:checked::before  {
```

Comment:
Two colons for before element(::before);

Reply:
hey, thank you for reviewing my code I have made all those changes that were pointed out by you